[embree3] Unknown CMake command "find_dependency"

**Host Environment**

- OS: windows 11
- Compiler: vs2019

**To Reproduce**

Steps to reproduce the behavior:

vcpkg install libigl[embree]:x64-windows

**Failure logs**
```
E:\vcpkg>vcpkg install libigl[embree]:x64-windows
Computing installation plan...
The following packages will be built and installed:
    libigl[core,embree]:x64-windows -> 2.3.0#2
Detecting compiler hash for triplet x64-windows...
-- Automatically setting HTTP(S)_PROXY environment variables to "127.0.0.1:7890".
Restored 0 package(s) from C:\Users\Administrator\AppData\Local\vcpkg\archives in 559 us. Use --debug to see more details.
Installing 1/1 libigl:x64-windows...
Building libigl[core,embree]:x64-windows...
-- Using cached libigl-libigl-e60423e28c86b6aa2a3f6eb0112e8fd881f96777.tar.gz.
-- Cleaning sources at E:/vcpkg/buildtrees/libigl/src/d881f96777-73695629ab.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source E:/vcpkg/downloads/libigl-libigl-e60423e28c86b6aa2a3f6eb0112e8fd881f96777.tar.gz
-- Applying patch fix-dependency.patch
-- Applying patch install-extra-headers.patch
-- Applying patch fix-config.patch
-- Using source at E:/vcpkg/buildtrees/libigl/src/d881f96777-73695629ab.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: E:/vcpkg/downloads/tools/cmake-3.25.1-windows/cmake-3.25.1-windows-i386/bin/cmake.exe E:/vcpkg/buildtrees/libigl/src/d881f96777-73695629ab.clean -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=E:/vcpkg/packages/libigl_x64-windows/debug -DFETCHCONTENT_FULLY_DISCONNECTED=ON -DLIBIGL_WITH_EMBREE=ON -DLIBIGL_WITH_OPENGL=OFF -DLIBIGL_WITH_OPENGL_GLFW=OFF -DLIBIGL_WITH_OPENGL_GLFW_IMGUI=OFF -DLIBIGL_WITH_XML=OFF -DLIBIGL_WITH_CGAL=OFF -DLIBIGL_BUILD_PYTHON=OFF -DLIBIGL_EXPORT_TARGETS=ON -DLIBIGL_USE_STATIC_LIBRARY=OFF -DLIBIGL_WITH_COMISO=OFF -DLIBIGL_WITH_TETGEN=OFF -DLIBIGL_WITH_TRIANGLE=OFF -DLIBIGL_WITH_PREDICATES=OFF -DLIBIGL_BUILD_TUTORIALS=OFF -DLIBIGL_WITH_PNG=OFF -DLIBIGL_BUILD_TESTS=OFF -DPYTHON_EXECUTABLE= "-DCMAKE_MAKE_PROGRAM=D:/Program Files/Microsoft Visual Studio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -DBUILD_SHARED_LIBS=ON -DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=E:/vcpkg/scripts/toolchains/windows.cmake -DVCPKG_TARGET_TRIPLET=x64-windows -DVCPKG_SET_CHARSET_FLAG=ON -DVCPKG_PLATFORM_TOOLSET=v142 -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE -DCMAKE_VERBOSE_MAKEFILE=ON -DVCPKG_APPLOCAL_DEPS=OFF -DCMAKE_TOOLCHAIN_FILE=E:/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON -DVCPKG_CXX_FLAGS= -DVCPKG_CXX_FLAGS_RELEASE= -DVCPKG_CXX_FLAGS_DEBUG= -DVCPKG_C_FLAGS= -DVCPKG_C_FLAGS_RELEASE= -DVCPKG_C_FLAGS_DEBUG= -DVCPKG_CRT_LINKAGE=dynamic -DVCPKG_LINKER_FLAGS= -DVCPKG_LINKER_FLAGS_RELEASE= -DVCPKG_LINKER_FLAGS_DEBUG= -DVCPKG_TARGET_ARCHITECTURE=x64 -DCMAKE_INSTALL_LIBDIR:STRING=lib -DCMAKE_INSTALL_BINDIR:STRING=bin -D_VCPKG_ROOT_DIR=E:/vcpkg -D_VCPKG_INSTALLED_DIR=E:/vcpkg/installed -DVCPKG_MANIFEST_INSTALL=OFF
    Working Directory: E:/vcpkg/buildtrees/libigl/x64-windows-dbg
    Error code: 1
    See logs for more information:
      E:\vcpkg\buildtrees\libigl\config-x64-windows-dbg-CMakeCache.txt.log
      E:\vcpkg\buildtrees\libigl\config-x64-windows-dbg-out.log
      E:\vcpkg\buildtrees\libigl\config-x64-windows-dbg-err.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:263 (vcpkg_execute_required_process)
  ports/libigl/portfile.cmake:25 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)
```

config-x64-windows-dbg-err.log
```
CMake Error at E:/vcpkg/installed/x64-windows/share/embree/embree-config.cmake:54 (find_dependency):
  Unknown CMake command "find_dependency".
Call Stack (most recent call first):
  E:/vcpkg/scripts/buildsystems/vcpkg.cmake:843 (_find_package)
  cmake/libigl.cmake:300 (find_package)
  CMakeLists.txt:59 (include)
```

--- a/ports/embree3/portfile.cmake
+++ b/ports/embree3/portfile.cmake
@@ -53,7 +53,7 @@ vcpkg_cmake_configure(
         -DEMBREE_TUTORIALS=OFF
         -DEMBREE_STATIC_RUNTIME=${EMBREE_STATIC_RUNTIME}
         -DEMBREE_STATIC_LIB=${EMBREE_STATIC_LIB}
-		-DEMBREE_INSTALL_DEPENDENCIES=OFF
+        -DEMBREE_INSTALL_DEPENDENCIES=OFF
 )
 
 vcpkg_cmake_install()
@@ -64,12 +64,12 @@ set(config_file "${CURRENT_PACKAGES_DIR}/share/embree/embree-config.cmake")
 file(READ "${config_file}" contents)
 string(REPLACE "SET(EMBREE_BUILD_TYPE Release)" "" contents "${contents}")
 string(REPLACE "/../../../" "/../../" contents "${contents}")
-string(REPLACE "FIND_PACKAGE" "find_dependency" contents "${contents}")
-string(REPLACE "REQUIRED" "" contents "${contents}")
+string(REPLACE "FIND_PACKAGE" "include(CMakeFindDependencyMacro)\n  find_dependency" contents "${contents}")
+string(REPLACE "REQUIRED" "COMPONENTS" contents "${contents}")
 string(REPLACE "/lib/cmake/embree-${VERSION}" "/share/embree" contents "${contents}")
 
 if(NOT VCPKG_BUILD_TYPE)
-	string(REPLACE "/lib/embree3.lib" "$<$<CONFIG:DEBUG>:/debug>/lib/embree3.lib" contents "${contents}")
+    string(REPLACE "/lib/embree3.lib" "$<$<CONFIG:DEBUG>:/debug>/lib/embree3.lib" contents "${contents}")
 endif()
 file(WRITE "${config_file}" "${contents}")
 

--- a/ports/embree3/vcpkg.json
+++ b/ports/embree3/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "embree3",
   "version": "3.13.5",
+  "port-version": 1,
   "description": "High Performance Ray Tracing Kernels.",
   "homepage": "https://github.com/embree/embree",
   "license": "Apache-2.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2194,7 +2194,7 @@
     },
     "embree3": {
       "baseline": "3.13.5",
-      "port-version": 0
+      "port-version": 1
     },
     "enet": {
       "baseline": "1.3.17",

--- a/versions/e-/embree3.json
+++ b/versions/e-/embree3.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "dcd33aea68c2058b380f0617fe5524a3939c3ac7",
+      "version": "3.13.5",
+      "port-version": 1
+    },
+    {
       "git-tree": "c4200bba761b3e04d73761b712080ade013f8763",
       "version": "3.13.5",
       "port-version": 0

